lib/ansible/plugins/filter/win_splitdrive.yml

```diff
@@ -13,17 +15,27 @@ DOCUMENTATION:
 
 EXAMPLES: |
 
-  # To get the last name of a file Windows path, like ['C', '\Users\asdf\foo.txt'] out of 'C:\Users\asdf\foo.txt'
+  # To get the last name of a file Windows path, like ['C:', '\\Users\\asdf\\foo.txt'] out of 'C:\Users\asdf\foo.txt'
```

Comment:
Within YAML the backslashes in single quoted strings do not need to be escaped.

Thus `'\\Users\\asdf\\foo.txt'` can be changed back to `'\Users\asdf\foo.txt'`

Source: https://docs.ansible.com/ansible/latest/os_guide/windows_usage.html#yaml-style